### PR TITLE
duplicate info about 3DES

### PR DESCRIPTION
**Pull request by BasWassenaar**
Removed line 40, same info named on (former) line 43. 
Also made it more clear that the signature algorithm should not be SHA1

### CODE REVIEW

File: azure-stack/operator/azure-stack-pki-certs.md [hunk 1]
```diff
@@ -37,8 +37,7 @@ The following list describes the general certificate issuance, security, and for
 ::: moniker-end
 - The use of self-signed certificates aren't supported.
 - For deployment and rotation, you can either use a single certificate covering all name spaces in the certificate's Subject Name and Subject Alternative Name (SAN) fields OR you can use individual certificates for each of the namespaces below that the Azure Stack Hub services you plan to utilize require. Both approaches require using wild cards for endpoints where they're required, such as **KeyVault** and **KeyVaultInternal**.
-- The certificate's PFX Encryption should be 3DES.
-- The certificate signature algorithm shouldn't be SHA1.
+- The certificate signature algorithm should **not** be SHA1.
```
Comment (PatAltimore) [inline azure-stack/operator/azure-stack-pki-certs.md:40]: Can you remove this change? I'd like to keep the original wording.

**Comment by PatAltimore** [comment]
Hi @BasWassenaar,

Thanks for the fix to remove the duplicate requirement. I'd like to keep the original wording...

- The certificate signature algorithm shouldn't be SHA1.

Could you make the change to your branch? Once you update the fix, I'll approve it to get merged.

Thanks again,
Pat

**Comment by PRMerger6** [comment]
@BasWassenaar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PatAltimore** [comment]
Hi @BasWassenaar,

Thanks! Looks good.

Pat
#sign-off

**Comment by BasWassenaar** [comment]
Thanks Pat. I'm kinda new to this. So learning a lot along the way. :-)

**Comment by PatAltimore** [comment]
Hi @BasWassenaar,

Thanks again! We appreciate the feedback and contributions.  I approved this change and it will get merged and published soon.

Thanks!
Pat

**Pull request merged**